.github/workflows/style-check.yaml

```diff
@@ -18,8 +18,9 @@ jobs:
 
       - name: Clang Format
         run: |
-          brew install clang-format
-          PATH="$(brew --prefix clang-format)/bin:${PATH}"
+          pip install --upgrade clang-format
+          which clang-format
+          clang-format --version
```

Comment:
Is this just diagnostics?

Reply:
See https://github.com/tenzir/vast/pull/2598#discussion_r983815089